lib/utils/import/text-files/index.ts

```diff
@@ -17,12 +17,18 @@ class TextFileImporter extends EventEmitter {
     let lastFileName = '';
 
     if (!filesArray) {
-      this.emit('status', 'error', 'No text files to import.');
+      this.emit('status', 'error', 'No files to import.');
       return;
     }
 
     const importTextFile = (file) => {
-      if (!file || !endsWith(file.name.toLowerCase(), '.txt')) {
+      if (
+        !file ||
+        !(
+          endsWith(file.name.toLowerCase(), '.txt') ||
+          endsWith(file.name.toLowerCase(), '.md')
+        )
+      ) {
```

Comment:
if we want we can combine these tests into a simple RegExp pattern. I like the pattern form and how it emphasizes the match over the transform. take it or leave it

```ts
if (!file || !/\.(txt|md)$/.test(file.name.toLowerCase())) { … }
```

add in optional-chaining…
```ts
if (!/\.(txt|md)$/.test(file?.name.toLowerCase() ?? '')) { … }
```

give it a name…
```ts
const hasAllowableName = /\.(txt|md)$/.test(file?.name.toLowerCase() ?? '');
if (!hasAllowableName) { … }
```

Reply:
Done